_chapters/chapter_13.md

```diff
@@ -757,7 +803,7 @@ end
       end
       ```
 
-      First, we'll remove the user_id we saved in the session by setting `session[:user_id]` to nil. Then, we'll show a friendly message confirming the user has been logged out. Finally, we'll redirect the user to the `root_path`.
+      The first line of this method removes the user_id we saved in the session by setting `session[:user_id]` to nil. The second line show a friendly message confirming the user has been logged out. The final line redirects the user to the `root_path`.
```

Comment:
I'm gonna try rewording this...

At the beginning of the method we'll remove the user's id from the session by setting `session[:user_id]` to `nil`. Then, we'll show a friendly message confirming the user has been logged out. Finally, we'll redirect the user to the `root_path`.

I don't feel to strongly about this. I like doing "first"..."then"..."finally", but that's just personal preference.